packages/expo-video/src/VideoView.tsx

```diff
@@ -24,15 +33,15 @@ export class VideoView extends PureComponent<VideoViewProps> {
   /**
    * Enters Picture in Picture (PiP) mode. Throws an exception if the device does not support PiP.
    * > **Note:** Only one player can be in Picture in Picture (PiP) mode at a time.
-   * @platform ios 14+
+   * @platform android, ios 14+
```

Comment:
```suggestion
   * @platform android
   * @platform ios 14+
```

Currently, to mark method correctly as supported by multiple platforms, we need to use separate `@platform` tag:
* https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#general-approach